changelog.md

```diff
@@ -1,3 +1,36 @@
+# 3.8.0 / 2021-04-27
+
+### New Features
+
+- Content can be added to personnal favourites
+- Add and comment a publication
+- Home/login page can be customizable 
+- Preview of links can be genearated in comment
+- Be able to fold the subspaces in the sidebar
+- New space dashboard
+- Possibilities to modify comments
+- New navigation links to user profile
+- Add button "open the gallery" on the content page
+- Allow to open file uploaded in a comment as a content
+- Rich text available in dashboard description
```

Comment:
```suggestion
- Content can be added to personal favourites
- Add and comment a publication
- The Home/login page can be customized 
- Links in comments are now previewed when possible
- Subspaces can be folded in the sidebar
- the dashboard has been revamped
- Comments can be edited and removed
- User profile can now be reached from places where the user has posted something 
- There is a new button on the content page to open the gallery
- files uploaded in a comment can now be opened as a content
- The space description is now editable using a rich text editor
```

Reply:
@raphj I update your suggestion with some details (to not have multiples suggestions for the same thing)

Reply:
@raphj I update your suggestion with some details (to not have multiples suggestions for the same thing)

Reply:
good call! :-)

Reply:
done